public/src/managers/GRPCManager.ts

```diff
@@ -190,9 +192,11 @@ export class GrpcManager {
         return this.grpcSend<Void>(this.agService.updateGroup, grp);
     }
 
-    public deleteGroup(groupid: number): Promise<IGrpcResponse<Void>> {
-        const request = new RecordRequest();
-        request.setId(groupid);
+    public deleteGroup(groupid: number, courseID: number, withRepo: boolean): Promise<IGrpcResponse<Void>> {
```

Comment:
Should all methods always have `courseID` first in the argument list.

Reply:
Should they? More than a half methods in this file have course ID as a second argument, but it is easy to fix.

Reply:
I think we should be consistent throughout the codebase that we always have `courseID` and other fields in the same order, if they are used. This is important when inputs are the same type. But especially important for API methods.
However, I will open a separate issue on this, so that we can do accept this one.